core/src/main/scala/kafka/coordinator/transaction/TransactionMetadata.scala

```diff
@@ -329,8 +347,12 @@ private[transaction] class TransactionMetadata(val transactionalId: String,
       case _ => txnStartTimestamp
     }
 
-    prepareTransitionTo(Ongoing, producerId, producerEpoch, lastProducerEpoch, txnTimeoutMs,
-      (topicPartitions ++ addedTopicPartitions).toSet, newTxnStartTimestamp, updateTimestamp, clientTransactionVersion)
+    prepareTransitionTo(
+      state = Ongoing,
+      topicPartitions = (topicPartitions ++ addedTopicPartitions),
+      txnStartTimestamp = newTxnStartTimestamp,
+      txnLastUpdateTimestamp = updateTimestamp
```

Comment:
should we include clientTransactionVersion here since it is now passed to this method too?